src/helpers/article.js

```diff
@@ -169,6 +170,25 @@ const extractArticles = result => result.reduce((accumulator, entry) => {
   return accumulator;
 }, []);
 
+/**
+ *converts comments array to include the net vote count
+ *
+ * @function
+ *
+ * @param {Array} comments - comment array to convert
+ *
+ * @returns {Array} - returns an array of comments with net vote count
+ */
+export const getVoteCount = comments => (comments.map((comment) => {
+  let voteCount = 0;
+  comment.CommentVotes.forEach(
+    (commentVote) => {
+      voteCount = (commentVote.dataValues.vote) ? voteCount += 1 : voteCount -= 1;
+    }
+  );
+  return { ..._.omit(comment.dataValues, ['CommentVotes']), voteCount };
```

Comment:
Great job @Bastieno so far. However, I do not understand why you would install a dependency 'Lodash' just to do this task of removing an object property that we have been achieving using object destructuring and the rest operator (...).
I think we need to keep our dependencies at a minimum.
You can check out how to implement this using the 'rest' operator as used in previous controllers and helpers.

Reply:
Oh @successgilli, thank you so much for catching this. I agree with him @Bastieno, please use the in-built Javascript functionality. We do not need lodash. Thanks

Reply:
> Great job @Bastieno so far. However, I do not understand why you would install a dependency 'Lodash' just to do this task of removing an object property that we have been achieving using object destructuring and the rest operator (...).
> I think we need to keep our dependencies at a minimum.
> You can check out how to implement this using the 'rest' operator as used in previous controllers and helpers.

My bad. I thought of using object destructuring but decided against it because
I thought that doing this `const { CommentVotes, ...commentData } = comment.dataValues;` and not using the `CommentVotes` that eslint will flag it as an unused variable.

I guess I should have tried it before jumping to a conclusion.